lib/orogen/gen/imports.rb

```diff
@@ -283,6 +283,12 @@ def task_context(name, &block) # :nodoc:
                 task.external_definition = true
                 task
             end
+            
+            def test_task_context(name, &block)
+                if test_support
+                    task_context("Test::" + name, &block)
```

Comment:
So far we use snake_case for namespaces. I would really prefer 'test' without a uppercase T